addons/project_purchase/models/project_project.py

```diff
@@ -12,36 +12,36 @@ class Project(models.Model):
 
     purchase_orders_count = fields.Integer('# Purchase Orders', compute='_compute_purchase_orders_count', groups='purchase.group_purchase_user', export_string_translation=False)
 
-    @api.depends('analytic_account_id')
     def _compute_purchase_orders_count(self):
-        data = self.env['purchase.order.line']._read_group(
-            [('analytic_distribution', 'in', self.analytic_account_id.ids)],
-            ['analytic_distribution'],
-            ['__count'],
-        )
-        data = {int(account_id): order_count for account_id, order_count in data}
+        purchase_count_per_project = {
+            project.id: count
+            for project, count in self.env['purchase.order']._read_group(
+                [('project_id', 'in', self.ids)],
+                ['project_id'], ['__count'],
+            )
+        }
```

Comment:
Maybe we could do that:
```suggestion
        purchase_count_per_project = dict(
            self.env['purchase.order']._read_group(
                [('project_id', 'in', self.ids)],
                ['project_id'], ['__count'],
            )
        )
```